core/templates/dev/head/pages/preferences/PreferencesSpec.js

```diff
@@ -54,5 +54,15 @@ describe('Preferences Controller', function() {
       $httpBackend.flush();
       expect(scope.canReceiveFeedbackMessageEmail).toBe(true);
     });
+
+    it('should map SUPPORTED_AUDIO_LANGUAGES correctly to ' +
+       'AUDIO_LANGUAGE_CHOICES to support select2 plugin',
+    function() {
+      var numberOfAudioLanguageChoices = scope.AUDIO_LANGUAGE_CHOICES.length;
+      for (var index = 0; index < numberOfAudioLanguageChoices; index++) {
```

Comment:
Here, add an assert to verify that the number of audio language choices is > 0 (to guard against possible skew between dev and test envs).

Reply:
Done!